packages/docs/src/commands/mutations/core-editing.mutation.ts

```diff
@@ -34,6 +34,8 @@ export interface IRichTextEditingMutationParams extends IMutationCommonParams {
     // Do you need to compose the undo and redo of history, and compose of the change states.
     debounce?: boolean;
     options?: { [key: string]: boolean };
+    // Whether this mutation is from a sync operation.
```

Comment:
What's sync operation?

Reply:
Mark this mutation is synced from other editor, like cell-editor to formula-bar-editor.

Reply:
> Mark this mutation is synced from other editor, like cell-editor to formula-bar-editor.

Mark this mutation is synced from other editor, like cell-editor to formula-bar-editor.